charts/promitor-agent-scraper/README.md

```diff
@@ -62,6 +62,7 @@ The following table lists the configurable parameters of the Promitor chart and
 | `metricDefaults.aggregation.interval`  | Default interval which defines over what period measurements of a metric should be aggregated | `00:05:00`            |
 | `metricDefaults.scraping.schedule`  | Cron expression that controls the fequency in which all the configured metrics will be scraped from Azure Monitor | `*/5 * * * *`            |
 | `metrics`  | List of metrics to scrape configured following the [metric declaration docs](https://promitor.io/configuration/metrics/) |        |
+| `resources`  | Pod resource requests & limits |    `{}`    |
```

Comment:
I'd split this actually into:
- `resources.limits.cpu` 
- `resources.limits.memory` 
- `resources.requests.cpu` 
- `resources.requests.memory`

Where we provide defaults for limits, but not for requests.

Reply:
Fair enough, less work for us 🤷‍♂️